protocols/generic_ngs_prep/README.md

```diff
@@ -0,0 +1,110 @@
+* `Number of Samples`: Enter total number of samples in the protocol run. **Note: Because both pipettes are 8-channels, the number of samples should be a multiple of 8 or close if possible, otherwise it will use up additional tips for an entire column.**
+
+---
+
+### Modules
+* [Temperature Module (GEN2)](https://shop.opentrons.com/collections/hardware-modules/products/tempdeck)
+* [Magnetic Module (GEN2)](https://shop.opentrons.com/collections/hardware-modules/products/magdeck)
+
+### Labware
+#### Plate options
+* [Bio-Rad 96 Well Plate 200 µL PCR](https://labware.opentrons.com/biorad_96_wellplate_200ul_pcr/)
+* [Opentrons 96 Well Aluminum Block with Generic PCR Strip 200 µL](https://labware.opentrons.com/opentrons_96_aluminumblock_generic_pcr_strip_200ul/)
+* [Opentrons 96 Well Aluminum Block with Bio-Rad Well Plate 200 µL](https://labware.opentrons.com/opentrons_96_aluminumblock_biorad_wellplate_200ul)
+* [Opentrons 96 Well Aluminum Block with NEST Well Plate 100 µL](https://labware.opentrons.com/opentrons_96_aluminumblock_nest_wellplate_100ul)
+* [NEST 96 Well Plate 100 µL PCR Full Skirt](https://labware.opentrons.com/nest_96_wellplate_100ul_pcr_full_skirt)
+
+#### Tips
+* [Opentrons 96 Filter Tip Rack 200 µL](https://labware.opentrons.com/opentrons_96_filtertiprack_200ul/)
+* [Opentrons 96 Filter Tip Rack 20 µL](https://labware.opentrons.com/opentrons_96_filtertiprack_20ul/)
+
+#### 12 well reservoir options
+* [NEST 12 Well Reservoir 15 mL](https://labware.opentrons.com/nest_12_reservoir_15ml/)
+* [USA Scientific 12 Well Reservoir 22 mL](https://labware.opentrons.com/usascientific_12_reservoir_22ml)
+
+#### 1 well reservoir options
+[Agilent 1 Well Reservoir 290 mL](https://labware.opentrons.com/agilent_1_reservoir_290ml)
+[Axygen 1 Well Reservoir 90 mL](https://labware.opentrons.com/axygen_1_reservoir_90ml)
+[NEST 1 Well Reservoir 195 mL](https://labware.opentrons.com/nest_1_reservoir_195ml)
+
+### Pipettes
+* [P20 Multichannel GEN2 Pipette](https://shop.opentrons.com/collections/ot-2-robot/products/8-channel-electronic-pipette?variant=5978988707869)
+* [P300 Multichannel GEN2 Pipette](https://shop.opentrons.com/collections/ot-2-robot/products/8-channel-electronic-pipette?variant=5984202489885)
+
+---
+
+### Deck Setup
+**Note: The deck setup changes throughout the course of the protocol. The initial calibration layout may differ from the final setup. Please follow the initial calibration setup and any instructions during the pauses.**
+
+#### Step 1
+* Slot 1: Temperature Module A + Sample Plate
+* Slot 3: Magnetic Module (Empty)
+* Slot 4: Temperature Module B + Reagent #1 Plate
+* Slot 6: 1 Well Reservoir (Ethanol)
+* Slot 7: Temperature Module C + Reagent 2 plate
+* Slot 8: Opentrons 200 uL Filter Tips
+* Slot 9: Opentrons 200 uL Filter Tips
+* Slot 10: Opentrons 20 uL Filter Tips
+* Slot 11: Opentrons 20 uL Filter Tips
+
+#### Step 2
+* Slot 1: Temperature Module A + Reagent reservoir (SPRI, Buffer I, Mastermix)
```

Comment:
The reservoir isn't "compatible" with the temperature module. Given this, reagents should potentially be split in this step with a 12-well reservoir in deck slot 2

Reply:
bump